runtimes/pythonrt/build.go

```diff
@@ -37,6 +37,13 @@ func isBuildFile(entry fs.DirEntry) bool {
 func (py *pySvc) Build(ctx context.Context, fsys fs.FS, path string, values []sdktypes.Symbol) (sdktypes.BuildArtifact, error) {
 	py.log.Info("build Python module", zap.String("path", path))
 
+	if path != "" {
```

Comment:
I don't see the value of sending a sub path. Is there a product requirement for that?

Reply:
the param is there and i think it's better than ignoring it.

Reply:
starlark is using it for what to build, so it's the same here now - just for directories.